src/Core/Prefab/GlobeView.js

```diff
@@ -71,27 +71,42 @@ export function createGlobeLayer(id, options = {}) {
 }
 
 /**
- * Creates the viewer Globe (the globe of iTowns).
- * The first parameter is the coordinates on wich the globe will be centered at the initialization.
- * The second one is the HTML div in wich the scene will be created.
+ * Creates a view of a globe.
+ *
  * @constructor
- * @example view = new GlobeView(viewer, positionOnGlobe);
- * // positionOnGlobe in latitude, longitude and altitude
- * @augments View
- * @param {HTMLDivElement} viewerDiv - Where to instanciate the Three.js scene in the DOM
- * @param {object} coordCarto
- * @param {object=} options - see {@link View}
+ *
+ * @example
+ * var viewerDiv = document.getElementById('viewerDiv');
+ * var position = new itowns.Coordinates('WGS84', 2.35, 48.8, 25e6);
+ * var view = new itowns.GlobeView(viewerDiv, position);
+ *
+ * @example
+ * var viewerDiv = document.getElementById('viewerDiv');
+ * var position = { longitude: 2.35, latitude: 48.8, altitude: 25e6 };
+ * var view = new itowns.GlobeView(viewerDiv, position);
+ *
+ * @param {HTMLDivElement} viewerDiv - Where to attach the view and display it
+ * in the DOM.
+ * @param {object|Coordinates} coordCarto - An object containing three
```

Comment:
This could also be [`cameraTransformOptions`](https://github.com/iTowns/itowns/blob/master/src/utils/CameraUtils.js#L322). 
it would also be necessary to remove the ambiguity between the camera position and the center of the view, because they can be different.

Reply:
Updated, WDYT